packages/headless/src/features/facets/facet-set/facet-set-slice.test.ts

```diff
@@ -200,6 +226,20 @@ describe('facet-set slice', () => {
       expect(finalState[id].freezeCurrentValues).toBe(true);
     });
 
+    it('does not set #freezeCurrentValues to true when toggled value does not exists', () => {
```

Comment:
```suggestion
    it('does not set #freezeCurrentValues to true when toggled value does not exist', () => {
```

Reply:
It would be helpful to group the tests for adding a new value together in a describe block:

```
describe('when the toggled value does not exist', () => {
  ...
})
```